hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientRatis.java

```diff
@@ -259,7 +259,7 @@ public XceiverClientReply watchForCommit(long index, long timeout)
       replyFuture.get(timeout, TimeUnit.MILLISECONDS);
     } catch (Exception e) {
       Throwable t = HddsClientUtils.checkForException(e);
-      LOG.warn("3 way commit failed ", e);
+      LOG.warn("3 way commit failed on pipeline " + pipeline, e);
```

Comment:
This can be changed to parameterized logging as 
LOG.warn("3 way commit failed on pipeline:{} ", pipeline, e);